src/views/addManager/addManager.js

```diff
@@ -81,7 +81,7 @@ export const AddManager = () => {
     }
 
     axios
-      .post(`/api/user/add`, body, makeAuthHeaders(context))
+      .post(`/api/user/invite`, body, makeAuthHeaders(context))
```

Comment:
The `/user/invite` endpoint has functionality to email the invited user. This is important information for the admin user to know, and it should be included in the Toast message (see line 93)

Reply:
Nice touch by @htharker42: `Toast('Property Manager Created Successfully, an invite email has been sent.', "success");`